editor/editor_export_godot3.cpp

```diff
@@ -2252,13 +2254,18 @@ Error EditorExportGodot3::_convert_script(const String &p_path, const String &p_
 
 		} while (count >= 1 && tries++ < 10);
 
+		if (mark_converted_lines && line != origline) {
+			// Add explanatory comment on the changed line
+			line += note;
```

Comment:
Initially I had a two line comment I wanted to put *before* the changed line, but I noticed that I would have to find out the indentation level, and didn't want to bother :P